src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/RazorProjectHost.cs

```diff
@@ -116,7 +118,7 @@ protected override Task InitializeCoreAsync(CancellationToken cancellationToken)
                 receiver,
                 initialDataAsNew: true,
                 suppressVersionOnlyUpdates: true,
-                ruleNames: new string[] { RazorGeneral.SchemaName, });
+                ruleNames: new string[] { Rules.RazorGeneral.SchemaName, Rules.RazorConfiguration.SchemaName, Rules.RazorExtension.SchemaName });
```

Comment:
Now we're listening for changes to the `RazorExtension` and `RazorConfiguration` itemgroups as well.